torch/csrc/jit/fuser/cpu/fused_kernel.cpp

```diff
@@ -16,9 +20,31 @@ namespace jit {
 namespace fuser {
 namespace cpu {
 
+#ifdef _MSC_VER
+static const std::string getTempPath() {
+  char lpTempPathBuffer[MAX_PATH];
+
+  DWORD dwRetVal = GetTempPath(
+      MAX_PATH, // length of the buffer
+      lpTempPathBuffer); // buffer for path
+
+  TORCH_CHECK(dwRetVal < MAX_PATH && dwRetVal != 0, "GetTempPath failed.");
+
+  return std::string(lpTempPathBuffer);
+}
+static const std::string temp_dir = getTempPath();
+static const std::string so_template = temp_dir + "pytorch_fuserXXXXXX.dll";
+static const std::string cpp_template = temp_dir + "pytorch_fuserXXXXXX.cpp";
+static const std::string check_exists_string = "where \"${program}\" > nul 2> nul";
+constexpr int so_suffix_len = 4;
```

Comment:
err... four?! XD

Reply:
Yes, the length of the suffix `.dll` is 4.

Reply:
You're right!